docs/configure-artifact-repository.md

```diff
@@ -360,36 +360,36 @@ artifacts:
       container: my-container-name
       blob: path/in/container
       # If a managed identity has been assigned to the machines running the
-      # workflow (e.g., https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
+      # workflow (for example, https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
       # then useSDKCreds should be set to true. The accountKeySecret is not required
       # and will not be used in this case.
       useSDKCreds: true  
 ```
 
-In addition to managed identities, Argo workflows also support authentication using access keys and SAS tokens.
+You can also authenticate with [Access Keys](#using-azure-access-keys) and [Shared Access Signatures (SAS)](#using-azure-shared-access-signatures-sas).
 
-### Using Azure access keys
+### Using Azure Access Keys
 
-1. Retrieve the blob service endpoint for the storage account. For example:
+1. Retrieve the blob service endpoint for the storage account.
 
     ```bash
     az storage account show -n mystorageaccountname --query 'primaryEndpoints.blob' -otsv
     ```
 
-2. Retrieve the access key for the storage account. For example:
+2. Retrieve the Access Key for the storage account.
 
     ```bash
     az storage account keys list -n mystorageaccountname --query '[0].value' -otsv
     ```
 
-3. Create a Kubernetes secret to hold the storage account key. For example:
+3. Create a Kubernetes secret to hold the storage account key.
 
     ```bash
     kubectl create secret generic my-azure-storage-credentials \
       --from-literal "account-access-key=$(az storage account keys list -n mystorageaccountname --query '[0].value' -otsv)"
     ```
 
-4. Configure `azure` artifact as follows in the yaml.
+4. Configure `azure` artifact as follows in the YAML.
```

Comment:
```suggestion
4. Configure an `azure` artifact:
```

- missing an article
- all the "YAML" descriptions are redundant, since the code block is YAML and does most of the talking (they predate your changes ofc and predate me being a contributor)